frontends/web/src/routes/account/add/add.tsx

```diff
@@ -58,46 +60,35 @@ class ManageAccount extends Component<Props, State> {
     public componentDidMount() {
         backendAPI.getSupportedCoins()
             // TEST with only 1 coin
-            // .then(() => (['btc']))
+            // .then(() => ([{coinCode: 'tbtc' as accountApi.CoinCode, name: 'Bitcoin Testnet', canAddAccount: true}]))
             .then((coins) => {
                 const onlyOneSupportedCoin = (coins.length === 1);
                 this.setState({
-                    // @ts-ignore
-                    coinCode: onlyOneSupportedCoin ? coins[0] : 'choose',
-                    supportedCoins: coins
+                    coinCode: onlyOneSupportedCoin ? coins[0].coinCode : 'choose',
+                    step: onlyOneSupportedCoin ? 'choose-name' : 'select-coin',
+                    supportedCoins: coins,
                 });
             })
-            .catch(console.error);
-    }
-
-    private getStep = () => {
-        const { step } = this.state;
-        if (this.onlyOneSupportedCoin()) {
-            return step === 0 ? 'choose-name' : 'success';
-        }
-        switch (step) {
-            case 0: return 'select-coin';
-            case 1: return 'choose-name';
-            case 2: return 'success';
-        }
+            .catch(error => this.showError(error));
```

Comment:
.catch() does nothing in production. Errors are always handled in the `then()` clause (success=false)

Reply:
ideally yes, but if there is a real error catch will catch it and we should display it to the user.

Reply:
as i said, `.catch()` after apiPost and apiGet is never called in production (in the Qt app). This line here will never execute, so you can safely remove it.

Reply:
done